Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -22,15 +22,11 @@ final class SettingsAccountViewController: UIViewController {
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    guard let messageBannerView = self.children.first as? MessageBannerViewController else {
-      fatalError("Couldn't instantiate MessageBannerViewController")
-    }
-
-    self.messageBannerView = messageBannerView
-
     self.tableView.dataSource = dataSource
     self.tableView.delegate = self
 
+    self.messageBannerViewController = configureMessageBannerViewController(on: self)
```

Comment:
Would we ever not pass `self` to this function? 🤔

If we added the `var` to the protocol then this could just be a call like:

```swift
self.configureMessageBannerViewController()
```

And then within that default implementation we could set the property.

Reply:
It felt a little side-effect-y when I initially did it this way; I thought it might be better to allow the caller to do what it wants with the configured `messageBannerViewController`. Setting it in the `configure` function would also require we make the function `mutating`. It seemed slightly more functional to have it take the parent view controller as an argument, and the subsequently configured `messageBannerViewController` as the return value.

Reply:
Cool, sounds good!